### PR TITLE
IA-664 polio display provinces

### PR DESCRIPTION
**Pull request by quang-le**
## Features

- Display Region boundaries on Scope map
- Add "Region" Column in Scope
- Table columns have a max width and will crop text on overflow. A tooltip display's the full name on hover
- Enable Region deselection from Scope table
- Added region to map tootltip (not shown on video)

## Changes
- Replaced `useGetRegionGeoJson` with `useGeoJson`, which can take 'DISTRICT' or 'REGION' as parameter
- Refactored `ScopeForm`'s `onSelectOrgUnit` for clarity
- Used separate `Pane`s in `MapComponent` to display districts and regions

https://user-images.githubusercontent.com/38907762/130103098-f550d083-8e09-4c54-adfa-a74544671bc7.mov

### CODE REVIEW

**Comment by quang-le** [comment]
## Update 20/08/2021

- add styling to cursor

**Comment by olethanh** [comment]
image
j'ai l'impression l'espace entre les colonnes est un peu trop grand, ca revient a la ligne alors qu'il reste plein d'espace

**Review by beygorghor — APPROVED** [review]

File: plugins/polio/js/src/components/MapComponent/MapComponent.js [hunk 1]
```diff
@@ -42,17 +53,42 @@ export const MapComponent = ({ onSelectShape, shapes, getShapeStyle }) => {
...
-                    <GeoJSON
-                        key={shape.id}
-                        data={shape.geo_json}
-                        style={() => getShapeStyle(shape)}
-                        onClick={() => onSelectShape(shape)}
-                    >
-                        <Tooltip>{shape.name}</Tooltip>
-                    </GeoJSON>
-                ))}
+            <Pane name="Regions">
```
[the remark this answers is not in the document]
Comment (quang-le) [reply plugins/polio/js/src/components/MapComponent/MapComponent.js]: > image
> I feel like the space between the column is a bit to big, it goes to the line when there is plenty of space left
If you refer to "CROSS RIVER" in the video, the name is actually too long for the column width, but the table wraps it at the space

**Comment by olethanh** [comment]
image
I added an image but it didn't work, for example here, North Horr shouldn't warp.

The actions seems to be center aligned, maybe we should also align the title?

**Comment by quang-le** [comment]
> image
> I added an image but it didn't work, for example here, North Horr shouldn't warp.
> 
> The actions seems to be center aligned, maybe we should also align the title?
> 
> maybe we could also decrease the font size.

Oh, I didn't realize the icon button's padding screwed the alignment. Nice catch. I'll make the font smaller as well

**Pull request merged**